src/supertux/menu/joystick_menu.cpp

```diff
@@ -187,31 +187,31 @@ JoystickMenu::refresh_menu_item(Controller::Control id)
     switch (hat_dir)
     {
       case SDL_HAT_UP:
-        name = "Hat Up";
+        name = _("Hat Up");
         break;
 
       case SDL_HAT_DOWN:
-        name = "Hat Down";
+        name = _("Hat Down");
         break;
 
       case SDL_HAT_LEFT:
-        name = "Hat Left";
+        name = _("Hat Left");
         break;
 
       case SDL_HAT_RIGHT:
-        name = "Hat Right";
+        name = _("Hat Right");
         break;
 
       default:
-        name = "Unknown hat_dir";
+        name = _("Unknown hat_dir");
```

Comment:
so it should throw some kind of exception. I'll see what I can whip up. what about the other strings above? are they fine as being translatable?